src/components/App.js

```diff
@@ -0,0 +1,69 @@
+import { useState, React, useEffect } from "react";
+import "../styles/App.css";
+import axios from "axios";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetailsBox from "./ForecastDetailsBox";
+import SearchForm from "./SearchForm";
+
+const App = () => {
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({ city: "", country: "" });
+  const [searchText, setSearchText] = useState("");
+
+  const [selectedDate, setSelectedDate] = useState(0);
+
+  const selectedForecast = forecasts.find(
+    (eforecast) => eforecast.date === selectedDate
+  );
+
+  const getForecast = () =>
+    // setSelectedDate,
+    // setForecasts,
+    // setLocation,
+    // searchText
+    {
```

Comment:
Looks like the comments here were notes to yourself, they should be removed by code review time
```suggestion
  const getForecast = () => {
```